h2integrate/converters/hydrogen/wombat_model.py

```diff
@@ -0,0 +1,109 @@
+        self.add_output("OpEx", val=0.0, units="USD/year", desc="Operational expenditure")
+        self.add_output(
+            "percent_hydrogen_lost",
+            val=0.0,
+            units="percent",
+            desc="Percent hydrogen lost due to O&M maintenance",
+        )
+        self.add_output(
+            "electrolyzer_availability", val=0.0, units=None, desc="Electrolyzer availability"
+        )
+
+    def compute(self, inputs, outputs):
+        super().compute(inputs, outputs)
+
+        # Ensure library_path is a Path object
+        library_path = self.config.library_path
+        if not isinstance(library_path, Path):
+            library_path = Path(library_path)
+        # Determine the correct library path: use as-is if absolute, else relative to this file
+        if library_path.is_absolute():
+            library_path = library_path
+        else:
+            library_path = Path(__file__).parents[3] / library_path
+
+        wombat_config = load_yaml(library_path, "electrolyzer.yml")
+        sim = Simulation(
+            library_path=library_path,
+            config=wombat_config,
+            random_seed=314,
+        )
+
+        sim.run(delete_logs=True, save_metrics_inputs=False)
+
+        scaling_factor = self.config.rating  # The baseline electrolyzer in WOMBAT is 1MW
+
+        # TODO: handle cases where the project is longer than one year.
+        # Do the project and divide by the project lifetime using sim.env.simulation_years
+
+        # Only support single electrolyzer systems for now, check the size of the returned values
+        availability_values = sim.metrics.operations[sim.metrics.electrolyzer_id].values
+        if availability_values.shape[1] > 1:
+            raise ValueError("Only single electrolyzer systems are supported at this time.")
+        availability = availability_values.flatten()
+
+        original_hydrogen_out = outputs["hydrogen_out"].copy()
+
+        # Adjust hydrogen_out by availability
+        hydrogen_out_with_availability = outputs["hydrogen_out"] * availability
+
+        # Update outputs["hydrogen_out"] with the available hydrogen
+        outputs["hydrogen_out"] = hydrogen_out_with_availability
+
+        # Compute total hydrogen produced (sum over the year)
+        outputs["total_hydrogen_produced"] = np.sum(hydrogen_out_with_availability)
+
+        # Compute percent hydrogen lost due to O&M maintenance
+        percent_hydrogen_lost = 100 * (
+            1 - outputs["total_hydrogen_produced"] / np.sum(original_hydrogen_out)
+        )
+
+        outputs["percent_hydrogen_lost"] = percent_hydrogen_lost
+
+        outputs["CapEx"] = self.config.electrolyzer_capex * self.config.rating * 1e3
+        outputs["OpEx"] = sim.metrics.opex("annual").iloc[0, 0] * scaling_factor
+        outputs["electrolyzer_availability"] = sim.metrics.time_based_availability(
+            "annual", "electrolyzer"
+        ).values[0]
```

Comment:
I have two non-blocking comments on this bit since this may be intentional.

1) `.squeeze()` will grab the values from 1x1 Series/DataFrame
2) More importantly, I noticed the "annual" value is now being grabbed, and it's worth pointing out that this is an annualized result, so the total for each simulated year. So, if you're seeking the annual average `.mean()` would likely be the most straightforward, though `metrics.<your-metric>("project") / sim.env.simulation_years` is the most accurate way to get the annual average as it accounts for incomplete years.

Reply:
One other note is that if a single year is being enforced for runtime, then I would recommend modifying the run call to stop the simulation at a year: `sim.run(until=8760)`.